shared/transforms/shared_constants.xslt

```diff
@@ -9,6 +9,7 @@
 <!-- abbreviated as references in the XCCDF-->
 <xsl:variable name="nist800-53uri">http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53r4.pdf</xsl:variable>
 <xsl:variable name="nist800-171uri">http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-171.pdf</xsl:variable>
+<xsl:variable name="cjisd-its-uri">https://www.fbi.gov/file-respository/cjis-security-policy-v5_5_20160601-2-1.pdf</xsl:variable>
```

Comment:
404 on that URL. Looks like the official URL actually has a typo in it:

https://www.fbi.gov/file-repository/cjis-security-policy-v5_5_20160601-2-1.pdf  -- in patch 
https://www.fbi.gov/file-respository/cjis-security-policy-v5_5_20160601-2-1.pdf -- official url

repository vs respository

Use https://www.fbi.gov/services/cjis/cjis-security-policy-resource-center instead?